### DISCUSSION TITLE
Give me a pointer, where to start translating

### DISCUSSION THREAD

**Discussion by PackmanDude** [MinersStudios/WhoMine#20] [Q&A] [answered]
Provide basic repo structure and explain how to translate descriptions to other languages, without affecting the original.

**Comment by nykonhrytsyshyn** [comment]
There is nothing to translate in this repository, and plugins don't yet support other languages

**Comment by PackmanDude** [comment]
> plugins don't yet support other languages

But when they will?

**Comment by PackmanDude** [comment, marked as answer]
Translations are fetched from https://github.com/MinersStudios/WMConfig and can be contributed to by submitting pull requests to the repository.
